pkg/object/objmetadata_test.go

```diff
@@ -481,7 +481,7 @@ func TestSetEquals(t *testing.T) {
 
 	for name, tc := range testCases {
 		t.Run(name, func(t *testing.T) {
-			actual := SetEquals(tc.setA, tc.setB)
+			actual := IsSetEqual(tc.setA, tc.setB)
```

Comment:
Please remove this. If you want this minor name change, please advocate for it in another PR.